src/engraving/internal/engravingconfiguration.cpp

```diff
@@ -88,19 +92,32 @@ void EngravingConfiguration::init()
     }
 
     static constexpr int ALL_VOICES_IDX = VOICES;
-    Settings::Key key("engraving", "engraving/colors/voiceAll");
-    settings()->setDefaultValue(key, Val(DEFAULT_VOICE_COLORS[ALL_VOICES_IDX].toQColor()));
-    settings()->setDescription(key, muse::qtrc("engraving", "All voices color").toStdString());
-    settings()->setCanBeManuallyEdited(key, true);
-    settings()->valueChanged(key).onReceive(this, [&](const Val& val) {
+    settings()->setDefaultValue(ALL_VOICES_COLOR, Val(DEFAULT_VOICE_COLORS[ALL_VOICES_IDX].toQColor()));
+    settings()->setDescription(ALL_VOICES_COLOR, muse::qtrc("engraving", "All voices color").toStdString());
```

Comment:
This can be written more concisely nowadays:
```suggestion
    settings()->setDescription(ALL_VOICES_COLOR, muse::trc("engraving", "All voices color"));
```
(similar below)